deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -12,24 +12,24 @@ metadata:
           },
           "spec": {
             "cpu": {
-              "isolated": "2-3",
+              "isolated": "1-3",
```

Comment:
Isolated and reserved must not overlap. Also for now, making nonIsolated the same as reserved is probably the best approach. In the future nonIsolated will be the same as all cpus - isolated and reserved will be subset of nonIsolated.

Reply:
ok, thx, fixed